umh.docs.umh.app/content/en/docs/production-guide/upgrading/upgrading.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Management Console Upgrades
+content_type: task
+description: |
+  This page describes how to perform the upgrades that are available for the Management Console.
+weight: 2000
+---
+
+<!-- overview -->
+
+Easily upgrade your UMH instance with the Management Console. This page offers clear, step-by-step instructions
+for a smooth upgrade process.
+
+## {{% heading "prerequisites" %}}
+
+Before proceeding with the upgrade of the Companion, ensure that you have the following:
+
+- A functioning UMH instance, verified as "online" and in good health.
+- A reliable internet connection.
+- Familiarity with the changelog of the new version you are upgrading to, especially to identify any breaking changes
+  or required manual interventions.
+
+## Management Companion
+
+Upgrade your UMH instance seamlessly using the Management Console. Follow these steps:
+
+### Navigate to the "Overview" Tab
+
+Access the Management Console's dashboard and click on the "Overview" tab.
```

Comment:
Remove this step and in the next just say something like "From the Overview tab..."